testinfra/conftest.py

```diff
@@ -26,19 +25,17 @@ def securedrop_import_testinfra_vars(hostname, with_header=False):
     Vars must be stored in `testinfra/vars/<hostname>.yml`.
     """
     filepath = os.path.join(os.path.dirname(__file__), "vars", hostname+".yml")
-    file_load = open(filepath, 'r')
+    hostvars = yaml.load(open(filepath, 'r'))
```

Comment:
Was not on my radar, :tophat: to you, sir. Will add. Gonna toss that `open` into a context manager while I'm in there.